pages/plugins/writing.md.erb

```diff
@@ -94,7 +97,20 @@ You can run the plugin linter with the following Docker command:
 docker run -it --rm -v "$PWD:/plugin:ro" buildkite/plugin-linter --id a-github-user/file-counter
```

Comment:
I think we should replace all of `## Step 3: Validate the plugin.yml` with:

````
## Step 3: Validate the plugin.yml

The [Buildkite Plugin Linter](https://github.com/buildkite-plugins/buildkite-plugin-linter) is an app that helps ensure your plugin is up-to-date and has all the files required to list it in the plugins directory. The app is available as a Docker image you can run on the command line, with a dedicated plugin, or with Docker Compose. We recommend you start by running the linter on the command line, and then include the dedicated plugin in the pipeline for your plugin.

### Run on the command line

You can run the plugin linter with the following Docker command:

```shell
docker run -it --rm -v "$PWD:/plugin:ro" buildkite/plugin-linter --id a-github-user/file-counter
```

### Run with the dedicated plugin

If your plugin has a Buildkite pipeline, you can add a step to lint it using the corresponding plugin:

```yml
  - label: ":shell: Lint"
    plugins:
      plugin-linter#v3.0.0:
        id: a-gihub-user/file-counter
```
{: codeblock-file=".buildkite/pipeline.yml"}

### Run with Docker Compose

If you want to run the linter using Docker Compose, you can add the following to a `docker-compose.yml` file:

```yml
services:
  lint:
    image: buildkite/plugin-linter
    command: ['--id', 'a-github-user/file-counter']
    volumes:
      - ".:/plugin:ro"
```
{: codeblock-file="docker-compose.yml"} 

You can then run the tests using the following command:

```shell
docker-compose run --rm lint
```
````

What do you think?

Reply:
Used as-is, except for the main heading that I changed to `Validate the plugin` (as the linter checks more things than just the `plugin.yml` file)